opm/parser/eclipse/EclipseState/Schedule/VFPProdTable.hpp

```diff
@@ -72,54 +72,8 @@ class VFPProdTable {
         ALQ_UNDEF, //< Undefined
         ALQ_INVALID
     };
-
-    /**
-     * Constructor
-     */
-    inline VFPProdTable() : m_table_num(-1),
-            m_datum_depth(-1),
-            m_flo_type(FLO_INVALID),
-            m_wfr_type(WFR_INVALID),
-            m_gfr_type(GFR_INVALID),
-            m_alq_type(ALQ_INVALID) {
-
-    }
-
-    /**
-     * Initializes objects from raw data. NOTE: All raw data assumed to be in SI units
-     * @param table_num VFP table number
-     * @param datum_depth Reference depth for BHP
-     * @param flo_type Specifies what flo_data represents
-     * @param wfr_type Specifies what wfr_data represents
-     * @param gfr_type Specifies what gfr_data represents
-     * @param alq_type Specifies what alq_data represents
-     * @param flo_data Axis for flo_type
-     * @param thp_data Axis for tubing head pressure
-     * @param wfr_data Axis for wfr_type
-     * @param gfr_data Axis for gfr_type
-     * @param alq_data Axis for alq_type
-     * @param data BHP to be interpolated. Given as a 5D array so that
-     *        BHP = data[thp][wfr][gfr][alq][flo] for the indices thp, wfr, etc.
-     */
-    void init(int table_num,
-            double datum_depth,
-            FLO_TYPE flo_type,
-            WFR_TYPE wfr_type,
-            GFR_TYPE gfr_type,
-            ALQ_TYPE alq_type,
-            const std::vector<double>& flo_data,
-            const std::vector<double>& thp_data,
-            const std::vector<double>& wfr_data,
-            const std::vector<double>& gfr_data,
-            const std::vector<double>& alq_data,
-            const array_type& data);
```

Comment:
The `init()` below can be avoided by using the constructor directly.  But this one, it looks like used to fill in some information based on the arguments. It is required by the 
```
 375     inline void initProperties() {
 376         //Initialize table
 377         table.init(table_ids[0],
 378                    1000.0,
 379                    Opm::VFPProdTable::FLO_OIL,
 380                    Opm::VFPProdTable::WFR_WOR,
 381                    Opm::VFPProdTable::GFR_GOR,
 382                    Opm::VFPProdTable::ALQ_UNDEF,
 383                    flo_axis,
 384                    thp_axis,
 385                    wfr_axis,
 386                    gfr_axis,
 387                    alq_axis,
 388                    data);
 389 
 390         //Initialize properties that use the table
 391         properties.reset(new Opm::VFPProdProperties(&table));
 392     }
```
in the `test_vfpproperties.cpp` on the simulator side.

Reply:
Either you can make another constructor out of this or you can make method `setProperties()` or still use the name `init()`. Basically, it looks like required to be able to set the content based on the above function arguments. If you have other suggestions to rewrite the `test`, it will also work.

Reply:
I will make another constructor.